recipes/qt/5.x.x/conanfile.py

```diff
@@ -1003,6 +1005,9 @@ def _create_plugin(pluginname, libname, type, requires):
         if self.options.qtnetworkauth:
             _create_module("NetworkAuth", ["Network"])
 
+        if self.options.qtx11extras:
```

Comment:
use `get_safe`, since this option is deleted in `configure()` if build machine is not Linux.